paper-tooltip.html

```diff
@@ -364,12 +345,51 @@
 
       },
 
+      _addListeners: function() {
+        if (this._target) {
+          this.listen(this._target, 'mouseenter', 'show');
+          this.listen(this._target, 'focus', 'show');
+          this.listen(this._target, 'mouseleave', 'hide');
+          this.listen(this._target, 'blur', 'hide');
+          this.listen(this._target, 'tap', 'hide');
+        }
+        this.listen(this, 'mouseenter', 'hide');
+      },
+
+      _findTarget: function() {
+        if (!this.manualMode)
+          this._removeListeners();
```

Comment:
this will do very little and is safe (at least with the current implementation of [`unlisten` in Polymer](https://github.com/Polymer/polymer/blob/edb59eb6283074ce54f374b30b884c9f9fcf9f5f/src/standard/events.html#L117-L128))

the alternative is to keep our own flag in this class as to whether we've added listeners onto |this|, but that seems duplicative.